_posts/newsletters/2020-11-30-newsletter.md

```diff
@@ -0,0 +1,115 @@
+
+<a name="leaders"></a>
+# **The 2nd International Leaders Workshop**
+
+The [2nd International Leaders Workshop](https://researchsoftware.org/2020-workshop.html) took place in September and attracted 39 participants from 13 countries on five continents, and included delegates from eight national or multinational RSE associations: [*de-RSE*](https://de-rse.org/en/), [*Society of RSE (UK)*](https://society-rse.org/), [*NL-RSE*](https://nl-rse.org/), [*US-RSE*](https://us-rse.org/), [*Nordic RSE*](https://nordic-rse.org/), [*African RSSE*](https://rsse-africa.sanbi.ac.za/login), [*beRSE*](https://www.be-rse.org/), and [*AUS/NZ RSE*](https://rse-aunz.github.io/). Even though they do not have formal RSE associations, individuals from Argentina, Colombia, France, Canada, and Spain also joined the workshop. The international attendance is thrilling and we are looking forward to more international collaboration! You can find more information about the workshop in the blog posts linked below in Interesting Reads.
+
+<a name="sc20"></a>
+# **RSE at SC20**
+
+We were excited to hold the inaugural Research Software Engineers in HPC Workshop (RSE-HPC-2020) as part of the SC20 conference this year!  The half-day, virtual event featured 18 speakers and had over 100 participants. Keynote addresses were given by RSE leaders from the UK and Germany on the history of the RSE movements in their countries.  The event also featured three lightning talks, and two panel discussions on the topics "Building RSE Teams and Groups" and “Supporting RSE Careers.”  We were glad to see the high level of interest in the talks and discussions, and we look forward to holding more of these workshops in the future.
+
+In addition to the RSE workshop, RSEs from the US and elsewhere led and participated in several other SC20 events.  The HPC community is giving increasing attention to the research software they run and the people who write it; this was evident in these events, and we hope to see the trend continue.
+
+![image alt text]({{ site.baseurl }}/assets/posts/2020-11-30-sc20-01.png "RSE-HPC-2020 screenshot from SC20")![image alt text]({{ site.baseurl }}/assets/posts/2020-11-30-sc20-02.png "RSE-HPC-2020 screenshot from SC20")
+
```

Comment:
this extra line leaves a blank line between the second and third images for me - I suggest we remove it

Reply:
I agree. I removed it.